zio/zeekio/builder.go

```diff
@@ -166,11 +167,31 @@ func (b *builder) appendPrimitive(typ zed.Type, val []byte) error {
 		b.buf = zed.AppendBool(b.buf[:0], v)
 	case zed.IDString:
 		// Zeek's enum type is aliased to string.
+		val = unescapeZeekString(val)
+		if !utf8.Valid(val) {
+			// Zeek has an unusual escaping model for non-valid UTF
+			// strings in their JSON integration: invalid bytes are
+			// formatted as the sequence '\' 'x' h h to indicate
```

Comment:
```suggestion
			// formatted as the sequence '\' 'x' to indicate
```

Reply:
There is probably a better way to say this but the point is subtle and I think dropping the salient piece of the encoding only makes it more confusing.